Adds Worker Id (#2722)
temporalio/documentation: docs-src/concepts/what-is-a-worker-identity.md

---
id: what-is-a-worker-identity
title: What is a Worker Identity?
sidebar_label: Worker Identity
description: A Worker Identity is an identifier for a Worker Client.
tags:
  - term
  - explanation
---

> Workers have an associated identifier that helps identify the specific Worker instance.

Reviewer: When we say "Worker instance", Is this at the Worker Process level or the Worker Entity level?

From below it seems like it is at the Process level - 
If so, it means one can have multiple Entities in the same process, but they would be identified as the same.

Can we clarify that?

Reviewer 2: In Core-based SDKs, `identity` can be specified per Client or per Worker (I mean instances of the SDK's Worker class), while in Go and Java, it can only be specified on Client. To keep documentation simple, it's probably better not include per-worker suffix in snippets.

> In most SDKs, the Worker Identity is referenced as `${process.pid}@${os.hostname()}`, which combines the Worker's process ID (`process.pid`) and the hostname of the machine running the Worker (`os.hostname()`).

Reviewer 2: ```suggestion
By default, Temporal SDKs set a Worker Identity to `${process.pid}@${os.hostname()}`, which combines the Worker's process ID (`process.pid`) and the hostname of the machine running the Worker (`os.hostname()`).
```

Reviewer 2: AFAIK, that default is applicable in all SDKs.
